api/handlers.go

```diff
@@ -747,6 +754,8 @@ func postContainersStart(c *context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	container.Config.HostConfig = hostConfig
```

Comment:
Thanks @vieux, this makes sense. Sorry I have to close this PR due to some git issues, let's move the conversation over to #1829.